waspc/data/Generator/templates/server/src/core/auth.js

```diff
@@ -48,7 +49,17 @@ export async function getUserFromToken(token) {
     }
   }
 
-  const user = await prisma.{= userEntityLower =}.findUnique({ where: { id: userIdFromToken } })
+  const user = await prisma.{= userEntityLower =}
+    .findUnique({
+      where: { id: userIdFromToken },
+      include: {
```

Comment:
So we are including auth and auth identities, are we ok with fetching so much stuff at once? I am on the side of fetching more if we are not sure, so I think this is likely a good decision for now, but still wanted to ask.

Reply:
I've commented elsewhere, but I think this is fine to start and we'll optimise down the line.